examples/access_traits/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include <ArborX.hpp>
+
+#include <Kokkos_Core.hpp>
+
+struct Data
+{
+  float *d_x;
+  float *d_y;
+  float *d_z;
+  int N;
+};
+
+struct Data2
+{
+  float *d_x;
+  float *d_y;
+  float *d_z;
+  float *d_r;
+  int N;
+};
+
+namespace ArborX
+{
+namespace Details
```

Comment:
Does a user-defined traits class make sense in the `Details` namespace or should it be elevated into the `ArborX` namespace?

Reply:
Fixed in #112